src/roadmap/common.py

```
@@ -57,6 +57,12 @@ def _fetch_rbac(fetch_url: str, fetch_headers: dict) -> dict:
     Takes url and headers as explicit arguments to ensure they are
     evaluated in the main thread before being passed to the worker thread.
     """
+    logger.info(f"RBAC request - URL: {fetch_url}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)